Makefile

```diff
@@ -59,7 +59,7 @@ sync:
 
 release:
 	# Bump the version
-	expr $$(cat BUILD) + 1 > BUILD
+	expr $$(cat BUILD) + 1 > BUILD_NUMBER
```

Comment:
I believe this line should read
`expr $$(cat BUILD_NUMBER) + 1 > BUILD_NUMBER`

Reply:
oh bummber! Yes .. I will fix it .. ta

Reply:
Fixed in https://github.com/pachyderm/python-pachyderm/pull/27/files Thanks @itssimon